src/engine/Engine.cpp

```diff
@@ -1192,6 +1200,42 @@ void Engine::performMILPSolverBoundedTightening()
     }
 }
 
+void Engine::performMILPSolverBoundedTighteningForSingleLayer( unsigned targetIndex )
+{
+    if ( _networkLevelReasoner && Options::get()->gurobiEnabled() && GlobalConfiguration::LP_TIGHTENING_AFTER_SPLIT )
+    {
+        _networkLevelReasoner->obtainCurrentBounds();
+
+        switch ( Options::get()->getMILPSolverBoundTighteningType() )
+        {
+        case MILPSolverBoundTighteningType::LP_RELAXATION:
+        case MILPSolverBoundTighteningType::LP_RELAXATION_INCREMENTAL:
+            _networkLevelReasoner->lpRelaxation( targetIndex );
+            break;
+
+        case MILPSolverBoundTighteningType::MILP_ENCODING:
+        case MILPSolverBoundTighteningType::MILP_ENCODING_INCREMENTAL:
```

Comment:
It looks like MILP_ENCODING_INCREMENTAL and LP_RELAXATION_INCREMENTAL are not implemented? Maybe also return in those cases?

Reply:
I changed so that it just returns.